lib/shopify-cli/messages/messages.rb

```diff
@@ -264,6 +258,19 @@ module Messages
             Usage: {{command:%s version}}
           HELP
         },
+
+        warning: {
+          development_version: <<~DEVELOPMENT,
+          {{*}} {{yellow:You are running a development version of the CLI at:}}
+              {{yellow:%s}}
+
+          DEVELOPMENT
+
+          shell_shim: <<~MESSAGE,
+          {{x}} {{red:The Shopify CLI is no longer available as a Git repository. It is now delivered as a package.
+            Please visit https://shopify.github.io/shopify-app-cli for installation instructions.}}
```

Comment:
```suggestion
          {{x}} {{red:This version of Shopify App CLI is no longer supported. You'll need to upgrade to continue using it.
          Please visit https://shopify.github.io/shopify-app-cli/upgrade/ for complete instructions. The upgrade typically takes a few minutes. }}
```

Just trying to make the call to action a little more direct here. It loses some technical detail but I think that should be deprioritized in favour of getting them to run the process.

The `https://shopify.github.io/shopify-app-cli/upgrade/` URL doesn't currently exist but I've added it as a redirect in #678.

Reply:
I actually only meant to comment, not approve 🤦‍♂️  But it does look good to me otherwise!

Reply:
I like the mention that the update is a quick process. Should we consider moving it into the first sentence, something like:
`This version of Shopify App CLI is no longer supported. You'll need to upgrade to continue using it, this process typically takes a few minutes.`

Reply:
Yeah, I'd be OK with that!

Reply:
I'll make it so. As for the implied promise, let's ask the Shopifolk who test this to keep track of how long it takes them to migrate, that should give us a better idea of the time investment needed.

Reply:
End result:

<img width="1045" alt="Screen Shot 2020-06-16 at 1 46 39 PM" src="https://user-images.githubusercontent.com/64600052/84809291-daafb880-afd7-11ea-99a6-8d08518313e1.png">

Reply:
Two things:

1. I'd prefer this text not be wholly red. Previously we've talked about not making error text entirely red, because the plain red color is hard to read if you have any color blindness (as I do!). (I know this change hasn't been made broadly yet, but I intend to go back and fix this.)

2. Let's put the URL on a separate line so it's easier to copy/paste/click, and use an underline for the URL. For that reason, I'd suggest some small tweaks to the text (and also, fix one run-on sentence… 🙂)

```
{{x}} This version of Shopify App CLI is no longer supported. You’ll need to upgrade to continue using it. This process typically takes a few minutes. Please visit this page for complete instructions:

{{underline:https://shopify.github.io/shopify-app-cli/upgrade/}}
```

Reply:
OK with me on all counts, except I would suggest maybe breaking before `Please visit this page for complete instructions:` as well. What do you think?

Reply:
Heh, I debated exactly that as well in my mind. Yep, sounds good.

Reply:
This would be the final version of the message (the color is much better indeed):
<img width="1078" alt="Screen Shot 2020-06-16 at 3 18 19 PM" src="https://user-images.githubusercontent.com/64600052/84818830-23ba3980-afe5-11ea-9782-3c67c6c55c81.png">

For comparison, this is what it would look like in a single line:
<img width="1076" alt="Screen Shot 2020-06-16 at 3 18 37 PM" src="https://user-images.githubusercontent.com/64600052/84818870-2fa5fb80-afe5-11ea-9fe4-0d13d7d9431c.png">